deployments/local-minio/deploy.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Name to give the docker network and stack. Update network instances in docker-compose.yml file if modified.
+networkname="cdr_local-minio"
+
+function wait_for_containers () {
+    waiting=`docker service ls | grep "cdr_local-minio_.*0/1 "`
```

Comment:
✏️     `waiting=$(docker service ls | grep "cdr_local-minio_.*0/1 ")`

Reply:
:+1: Updated the other instances, and also replaced all the spots I was using "cdr_local-minio" with $networkname as well.